.zappr.yaml

```diff
@@ -1,15 +1,20 @@
 approvals:
-  # PR needs at least 1 approval
-  minimum: 1
+  # PR needs at least 2 approvals
+  minimum: 2
   # approval = comment that matches this regex
   pattern: "^(:\\+1:|👍)$"
   # note that `from` is by default empty,
   # accepting any matching comment as approval
-  from:
-    # commenter must be either one of:
-    orgs:
-      # a member of norris organization
-      # (any org in here counts)
+  groups:
+   # there must be at least a technical approval from a member of team Norris
+   technical:
+    minimum: 1
+    from:
+     orgs:
       - norris
```

Comment:
They're also GH, I created the UX one this time (https://github.com/orgs/zalando/teams/ux-brand-solutions) and the Norris one somewhen in the past, and it was actually already in use (https://github.com/orgs/zalando/teams/norris)